.ruby-version

```diff
@@ -1 +1 @@
-2.6.6
+2.7.0
```

Comment:
Did you mean for this change to be in the PR?

Reply:
Oops, no, changing it now :)